packages/twenty-front/src/modules/action-menu/actions/record-actions/single-record/hooks/useFirstSelectedRecordId.ts

```diff
@@ -0,0 +1,18 @@
+import { contextStoreTargetedRecordsRuleComponentState } from '@/context-store/states/contextStoreTargetedRecordsRuleComponentState';
+import { useRecoilComponentValueV2 } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentValueV2';
+
+export const useFirstSelectedRecordId = () => {
```

Comment:
we should add contextStore in the naming 
useContextStoreFirstSelectedRecordId
also, maybe a selector to avoid re-renders here?